pages/search.vue

```diff
@@ -137,6 +140,15 @@ export default {
     async onKeywordChange(e) {
       const { colCount, searchQuery } = this;
       if (searchQuery.length === 0) {
+        // fetch featured image list if no search result
+        if (this.featuredImages.length === 0) {
+          try {
+            const { data } = await axios.get('/api/assets/featured/list');
+            this.featuredImages = this.sortImagesByHeight(data, colCount);
+          } catch (err) {
+            console.error(err); // eslint-disable-line no-console
+          }
+        }
```

Comment:
Suggest to refactor fetching logic to vuex action that is commonly shared with carousel